### ISSUE TITLE
Logic for `$excluded` variable cannot possibly work reliably because the second if statement might overwrite the first.

### ISSUE THREAD

**Issue by texnixe** [Werbschaft/kirby-upload-extended#1]
If you use a single if statement, you avoid this problem:
```
  if( !empty($excludeTemplates) || !empty($excludePages ) ) {
      $excluded = in_array( $file->page()->intendedTemplate(), $excludeTemplates ) || in_array( $file->page()->uid(), $excludePages );
    }
```

**Comment by Werbschaft** [comment]
I have now also thought for a while and have come to this solution that works after a few tests:

```php
if ( !empty($excludeTemplates) || !empty($excludePages) ) {		
  $excluded = true;
  if( !empty($excludeTemplates) ) {
    $excluded .= in_array( $file->page()->intendedTemplate(), $excludeTemplates );
  }
  if (!empty($excludePages) ) {
    $excluded .= in_array( $file->page()->uid(), $excludePages );
  }
}
```

But @texnixe what can I say, your solution is clearly better!!! I have unfortunately only seen after I have already started to test. I will fix!

**Comment by texnixe** [comment]
I'm sorry, but your code is not only overly complicated but will return a string (e.g '111'  if both conditions are true) instead of a boolean.

**Comment by Werbschaft** [comment]
OMG, you're right! I'm also a little ashamed to be honest! Thanks for your help 💛

**Comment by texnixe** [comment]
Fixed with ce9137fe4ac9123ed62e794e8b5e4f75ad02e034
